### DISCUSSION TITLE
Could not load python38.dll.

### DISCUSSION THREAD

**Discussion by Aloento** [pythonnet/pythonnet#1923] [Q&A]
### Environment

-   Pythonnet version: 3.0.0-rc4
-   Python version: 3.8.10
-   Operating System: Windows 11
-   .NET Runtime: .NET 6 / 7

### Details

-   Describe what you were trying to get done.

    When there is no Python already installed on the system.
    Trying to load `python-3.8.10-embed-amd64` 

-   What commands did you run to trigger this issue? If you can provide a
    [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
    this will help us understand the issue.

    `PythonEngine.PythonHome = path_to_embed_python38`
    https://github.com/Aloento/CraftVits/blob/0e791776deccc6e6428c0ba339ef97aa168db2b6/VitsCLI/PyLoc.cs#L10

-   If there was a crash, please include the traceback here.

```
Unhandled exception. System.AggregateException: One or more errors occurred. (The type initializer for 'Delegates' threw an exception.)
 ---> System.TypeInitializationException: The type initializer for 'Delegates' threw an exception.
 ---> System.DllNotFoundException: Could not load python38.dll.
 ---> System.ComponentModel.Win32Exception (126): The specified module could not be found.
   --- End of inner exception stack trace ---
   at Python.Runtime.Platform.WindowsLoader.Load(String dllToLoad)
   at Python.Runtime.Runtime.Delegates..cctor()
   --- End of inner exception stack trace ---
   at Python.Runtime.PythonEngine.set_PythonHome(String value)
```

If this system has Python38 installed, then no error.

**Comment by lostmsu** [comment]
You need to set `Runtime.PythonDLL`
